dev/tests/integration/testsuite/Magento/Newsletter/Controller/Manage/SaveTest.php

```diff
@@ -126,6 +125,7 @@ public function testUnsubscribeSubscribedCustomer(): void
     private function dispatchSaveAction(bool $isSubscribed): void
     {
         $this->_objectManager->removeSharedInstance(CustomerPlugin::class);
```

Comment:
should we remove the shared instance also here?

Reply:
No, the test will fail.